### DISCUSSION TITLE
Tasmota Zigbee Always Can not Start

### DISCUSSION THREAD

**Discussion by vensolezl** [arendst/Tasmota#22664] [Q&A]
I have a sonoff zb brigde pro device that was previously successfully flashed to tasmota firmware. 
The zigbee module in the device started normally, and I was able to brush in the latest tasmota version and successfully update the znp firmware.
Then zigbee in the module can not start normally, I looked up some information, but always can not find the possible problem.
Can someone help me tell me what the possible cause is? thanks!
The following is the startup log of the system:
![tasmota](https://github.com/user-attachments/assets/f4de8445-e652-46a4-9d11-008da8d64b74)

**Comment by s-hadinger** [comment]
Set log to `weblog 3` and report back logs

**Reply by vensolezl** [reply]
Thank you for your reply. The following is my log record.
-------------------------------------------------------------------------------------------
```
00:00:00.001 HDW: ESP32-D0WD-V3 v3.1 (PSRAM)
00:00:00.100 UFS: FlashFS mounted with 800 kB free
00:00:00.119 CFG: Loaded from File, Count 40
00:00:00.128 QPC: Count 1
00:00:00.129 CFG: CR 394/699, Busy 0
00:00:00.132 I2C: Bus1 using GPIO26(SCL) and GPIO25(SDA)
00:00:00.133 TYA: Active=0
00:00:00.135 ROT: Mode 1
00:00:00.195 BRY: Berry initialized, RAM used 3508 bytes
00:00:00.238 BRY: No 'preinit.be'
00:00:00.240 ZIG: Using Hardware Serial UART2
00:00:00.241 SRC: Restart
00:00:00.243 Project tasmota - Tasmota Version 14.4.1(release-zbbrdgpro)-3_1_0(2024-12-15T13:32:49)
00:00:00.245 ZIG: Loading Zigbee data
00:00:00.252 ZIG: No Zigbee device information
00:00:00.255 ZIG: No Zigbee device data
00:00:00.256 ZIG: ZbLoad '<internal_plugin>' loaded successfully
00:00:00.569 BRY: No 'autoexec.be'
00:00:00.844 WIF: Attempting connection...
00:00:01.370 WIF: Connecting to AP1 RAX50 Channel 8 BSSId 74:D0:2B:28:3D:5B in mode HT40 as tasmota-4BED4C-3404...
00:00:02.043 WIF: IPv4 10.0.0.111, mask 255.255.255.0, gateway 10.0.0.1
00:00:03.917 WIF: IPv6 Local fe80::e665:b8ff:fe4b:ed4c%st1
00:00:04.008 WIF: Connected
00:00:04.215 HTP: Web server active on tasmota-4BED4C-3404 with IP address 10.0.0.111
00:00:04.217 RSL: INFO1 = {"Info1":{"Module":"Sonoff Zigbee Pro","Version":"14.4.1(release-zbbrdgpro)","FallbackTopic":"cmnd/DVES_4BED4C_fb/","GroupTopic":"cmnd/tasmotas/"}}
00:00:04.231 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-4BED4C-3404","IPAddress":"10.0.0.111","IP6Global":"","IP6Local":"fe80::e665:b8ff:fe4b:ed4c%st1"}}
00:00:04.254 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":22}}
00:00:05.056 RTC: UTC 2024-12-18T23:59:33Z, DST 2024-03-31T02:00:00, STD 2024-10-27T03:00:00
00:59:33.001 RTC: Synced by NTP
00:59:35.869 QPC: Reset
00:59:35.917 WIF: IPv6 Global 2001:250:5403:420:e665:b8ff:fe4b:ed4c
00:59:36.868 RSL: STATE = {"Time":"2024-12-19T00:59:36","Uptime":"0T00:00:08","UptimeSec":8,"Heap":167,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":45},"Wifi":{"AP":1,"SSId":"RAX50","BSSId":"74:D0:2B:28:3D:5B","Channel":8,"Mode":"HT40","RSSI":100,"Signal":-34,"LinkCount":1,"Downtime":"0T00:00:04"}}
00:59:37.862 APP: Boot Count 22
00:59:38.864 CFG: Saved, Count 41, Bytes 4096
00:59:43.622 ZIG: rebooting ZNP device
00:59:46.874 ZIG: {"ZbZNPReceived":"4180000201020701"}
00:59:46.876 RSL: RESULT = {"ZbState":{"Status":1,"Message":"CCxxxx ZNP booted","RestartReason":"Power-up","MajorRel":2,"MinorRel":7}}
00:59:47.022 ZIG: checking device configuration
00:59:47.023 ZIG: ZbZNPSent 2102
00:59:47.074 ZIG: {"ZbZNPReceived":"6102020102070146D9340100"}
00:59:47.075 RSL: RESULT = {"ZbState":{"Status":50,"MajorRel":2,"MinorRel":7,"MaintRel":1,"Revision":20240710}}
00:59:47.077 ZIG: ZbZNPSent 2108600000
00:59:47.123 ZIG: {"ZbZNPReceived":"6108000155"}
00:59:47.124 ZIG: ZbZNPSent 2108870000
00:59:47.173 ZIG: {"ZbZNPReceived":"6108000100"}
00:59:47.174 ZIG: ZbZNPSent 2108830000
00:59:47.250 ZIG: {"ZbZNPReceived":"61080002FFFF"}
00:59:47.251 RSL: RESULT = {"ZbState":{"Status":2,"Message":"Resetting configuration"}}
00:59:47.255 ZIG: ZbZNPSent 21090300000103
00:59:47.273 ZIG: {"ZbZNPReceived":"610900"}
00:59:47.274 ZIG: ZbZNPSent 410000
00:59:48.923 ZIG: {"ZbZNPReceived":"4180000201020701"}
00:59:48.925 ZIG: ZbZNPSent 2109830000020100
00:59:48.973 ZIG: {"ZbZNPReceived":"610900"}
00:59:48.974 ZIG: ZbZNPSent 21092D00000800000000CCCCCCCC
00:59:49.023 ZIG: {"ZbZNPReceived":"610900"}
00:59:49.025 ZIG: ZbZNPSent 21098400000400080000
00:59:49.073 ZIG: {"ZbZNPReceived":"610900"}
00:59:49.074 ZIG: ZbZNPSent 21098700000100
00:59:49.123 ZIG: {"ZbZNPReceived":"610900"}
00:59:49.125 ZIG: ZbZNPSent 21096200001000004CED0000000000004CED00000000
00:59:49.173 ZIG: {"ZbZNPReceived":"610900"}
00:59:49.174 ZIG: ZbZNPSent 21096300000100
00:59:49.223 ZIG: {"ZbZNPReceived":"610900"}
00:59:49.225 ZIG: ZbZNPSent 210901010020FFFFFFFFFFFFFFFF5A6967426565416C6C69616E636530390000000000000000
00:59:49.273 ZIG: {"ZbZNPReceived":"610902"}
00:59:49.274 ZIG: ZbZNPSent 21098F00000101
00:59:49.323 ZIG: {"ZbZNPReceived":"610900"}
00:59:49.324 ZIG: ZbZNPSent 2107600001000100
00:59:49.373 ZIG: {"ZbZNPReceived":"610700"}
00:59:49.374 ZIG: ZbZNPSent 21096000000155
00:59:49.423 ZIG: {"ZbZNPReceived":"610900"}
00:59:49.424 ZIG: ZbZNPSent 2108620000
00:59:49.474 ZIG: {"ZbZNPReceived":"6108001000004CED0000000000004CED00000000"}
00:59:49.476 RSL: RESULT = {"ZbState":{"Status":3,"Message":"Configured, starting coordinator"}}
00:59:49.478 ZIG: ZbZNPSent 2F080100080000
00:59:49.523 ZIG: {"ZbZNPReceived":"6F0800"}
00:59:49.524 ZIG: ZbZNPSent 2F080000080000
00:59:49.573 ZIG: {"ZbZNPReceived":"6F0800"}
00:59:49.574 ZIG: ZbZNPSent 2F0504
00:59:52.424 ZIG: {"ZbZNPReceived":"6F0500"}
00:59:52.474 ZIG: {"ZbZNPReceived":"4F80010204"}
00:59:52.724 ZIG: {"ZbZNPReceived":"45C008"}
00:59:54.924 ZIG: {"ZbZNPReceived":"45C008"}
00:59:55.124 ZIG: {"ZbZNPReceived":"45C008"}
00:59:55.474 ZIG: {"ZbZNPReceived":"45C000"}
00:59:55.524 ZIG: {"ZbZNPReceived":"4F80080200"}
00:59:55.574 ZIG: {"ZbZNPReceived":"4F80080200"}
01:00:12.474 ZIG: timeout, goto label 99
01:00:12.476 RSL: RESULT = {"ZbState":{"Status":99,"Message":"Abort"}}
01:00:12.478 ZIG: Abort
01:00:12.479 ZIG: Stopping (99)
```

**Reply by vensolezl** [reply]
Here is the firmware I compiled myself, using Chinese Settings, adding Ethernet support based on Lan8720, and using Ebyte's E72 CC2652P module. The device also has the problem that zigbee cannot start normally, the following is the log record, please help me take a look together, thanks.
--------------------------------------------------------------------------------------------------
00:00:00.002 HDW: ESP32-D0WD-V3 v3.1 
00:00:00.010 UFS: FlashFS mounted with 4400 kB free
00:00:00.016 CFG: Loaded from File, 数量: 20
00:00:00.023 QPC: Count 1
00:00:00.024 CFG: CR 342/699, Busy 0
00:00:00.026 TYA: Active=0
00:00:00.028 ROT: Mode 1
00:00:00.045 CFG: No '*.autoconf' file found
00:00:00.048 BRY: Berry initialized, RAM used 3478 bytes
00:00:00.052 BRY: No 'preinit.be'
00:00:00.055 ZIG: Using Hardware Serial UART2
00:00:00.055 SRC: Restart
00:00:00.057 项目: tasmota - Tasmota 版本 14.3.0(tasmota)-3_1_0(2024-12-10T16:01:47)
00:00:00.058 ZIG: Loading Zigbee data
00:00:00.061 ZIG: No Zigbee device information
00:00:00.062 ZIG: No Zigbee device data
00:00:00.063 ZIG: ZbLoad '<internal_plugin>' loaded successfully
00:00:00.084 BRY: No 'autoexec.be'
00:00:00.260 WIF: 尝试连接...
00:00:01.000 WIF: 连接到 AP1 RAX50 Channel 8 BSSId 74:D0:2B:28:3D:5B 模式: HT40 名称: tasmota-E8FF34-7988...
00:00:01.042 WIF: IPv4 10.0.0.209, mask 255.255.255.0, gateway 10.0.0.1
00:00:02.992 WIF: 已连接
00:00:03.372 RTC: UTC 2024-12-19T00:29:10Z, DST 2024-03-31T02:00:00, STD 2024-10-27T03:00:00
01:29:10.001 RTC: Synced by NTP
01:29:10.011 HTP: Web 服务器地址: tasmota-E8FF34-7988 IP 地址: 10.0.0.209
01:29:10.065 WIF: IPv6 Local fe80::1206:1cff:fee8:ff34%st1
01:29:11.277 RSL: INFO1 = {"Info1":{"Module":"ESP32-DevKit","Version":"14.3.0(tasmota)","FallbackTopic":"cmnd/DVES_E8FF34_fb/","GroupTopic":"cmnd/tasmotas/"}}
01:29:11.280 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-E8FF34-7988","IPAddress":"10.0.0.209","IP6Global":"","IP6Local":"fe80::1206:1cff:fee8:ff34%st1"}}
01:29:11.281 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":10}}
01:29:13.981 QPC: Reset
01:29:14.437 WIF: IPv6 Global 2001:250:5403:420:1206:1cff:fee8:ff34
01:29:15.976 APP: 启动次数 10
01:29:15.981 RSL: STATE = {"Time":"2024-12-19T01:29:15","Uptime":"0T00:00:09","UptimeSec":9,"Heap":159,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":40},"Wifi":{"AP":1,"SSId":"RAX50","BSSId":"74:D0:2B:28:3D:5B","Channel":8,"Mode":"HT40","RSSI":94,"Signal":-53,"LinkCount":1,"Downtime":"0T00:00:03"}}
01:29:16.871 CFG: Saved, 数量: 21, 大小: 4096
01:29:18.784 HTP: 主菜单
01:29:21.860 HTP: 控制台
01:29:22.569 ZIG: rebooting ZNP device
01:29:22.821 HTP: 控制台
01:29:25.933 ZIG: {"ZbZNPReceived":"4180000201020701"}
01:29:25.934 RSL: RESULT = {"ZbState":{"Status":1,"Message":"CCxxxx ZNP booted","RestartReason":"Power-up","MajorRel":2,"MinorRel":7}}
01:29:26.082 ZIG: checking device configuration
01:29:26.083 ZIG: ZbZNPSent 2102
01:29:26.133 ZIG: {"ZbZNPReceived":"6102020102070146D9340100"}
01:29:26.134 RSL: RESULT = {"ZbState":{"Status":50,"MajorRel":2,"MinorRel":7,"MaintRel":1,"Revision":20240710}}
01:29:26.135 ZIG: ZbZNPSent 2108600000
01:29:26.182 ZIG: {"ZbZNPReceived":"6108000155"}
01:29:26.183 ZIG: ZbZNPSent 2108870000
01:29:26.232 ZIG: {"ZbZNPReceived":"6108000100"}
01:29:26.233 ZIG: ZbZNPSent 2108830000
01:29:26.282 ZIG: {"ZbZNPReceived":"61080002FFFF"}
01:29:26.284 RSL: RESULT = {"ZbState":{"Status":2,"Message":"Resetting configuration"}}
01:29:26.285 ZIG: ZbZNPSent 21090300000103
01:29:26.332 ZIG: {"ZbZNPReceived":"610900"}
01:29:26.333 ZIG: ZbZNPSent 410000
01:29:28.233 ZIG: {"ZbZNPReceived":"4180000201020701"}
01:29:28.233 ZIG: ZbZNPSent 2109830000020100
01:29:28.282 ZIG: {"ZbZNPReceived":"610900"}
01:29:28.283 ZIG: ZbZNPSent 21092D00000800000000CCCCCCCC
01:29:28.332 ZIG: {"ZbZNPReceived":"610900"}
01:29:28.333 ZIG: ZbZNPSent 21098400000400080000
01:29:28.382 ZIG: {"ZbZNPReceived":"610900"}
01:29:28.383 ZIG: ZbZNPSent 21098700000100
01:29:28.432 ZIG: {"ZbZNPReceived":"610900"}
01:29:28.433 ZIG: ZbZNPSent 210962000010000034FF00000000000034FF00000000
01:29:28.482 ZIG: {"ZbZNPReceived":"610900"}
01:29:28.483 ZIG: ZbZNPSent 21096300000100
01:29:28.532 ZIG: {"ZbZNPReceived":"610900"}
01:29:28.534 ZIG: ZbZNPSent 210901010020FFFFFFFFFFFFFFFF5A6967426565416C6C69616E636530390000000000000000
01:29:28.582 ZIG: {"ZbZNPReceived":"610902"}
01:29:28.583 ZIG: ZbZNPSent 21098F00000101
01:29:28.632 ZIG: {"ZbZNPReceived":"610900"}
01:29:28.633 ZIG: ZbZNPSent 2107600001000100
01:29:28.682 ZIG: {"ZbZNPReceived":"610700"}
01:29:28.683 ZIG: ZbZNPSent 21096000000155
01:29:28.732 ZIG: {"ZbZNPReceived":"610900"}
01:29:28.733 ZIG: ZbZNPSent 2108620000
01:29:28.783 ZIG: {"ZbZNPReceived":"61080010000034FF00000000000034FF00000000"}
01:29:28.784 RSL: RESULT = {"ZbState":{"Status":3,"Message":"Configured, starting coordinator"}}
01:29:28.786 ZIG: ZbZNPSent 2F080100080000
01:29:28.832 ZIG: {"ZbZNPReceived":"6F0800"}
01:29:28.833 ZIG: ZbZNPSent 2F080000080000
01:29:28.882 ZIG: {"ZbZNPReceived":"6F0800"}
01:29:28.883 ZIG: ZbZNPSent 2F0504
01:29:32.234 ZIG: {"ZbZNPReceived":"6F0500"}
01:29:32.284 ZIG: {"ZbZNPReceived":"4F80010204"}
01:29:32.534 ZIG: {"ZbZNPReceived":"45C008"}
01:29:34.784 ZIG: {"ZbZNPReceived":"45C008"}
01:29:35.084 ZIG: {"ZbZNPReceived":"45C008"}
01:29:35.634 ZIG: {"ZbZNPReceived":"45C000"}
01:29:35.684 ZIG: {"ZbZNPReceived":"4F80080200"}
01:29:35.734 ZIG: {"ZbZNPReceived":"4F80080200"}
01:29:52.236 ZIG: timeout, goto label 99
01:29:52.237 RSL: RESULT = {"ZbState":{"Status":99,"Message":"Abort"}}
01:29:52.239 ZIG: Abort
01:29:52.239 ZIG: Stopping (99)

**Reply by vensolezl** [reply]
Here are the Sonoff ZB Pro boards that I used and the boards that I designed myself, and strangely, neither can use Zibgee.
![e574f3fe04ac6db97703082325b55ee](https://github.com/user-attachments/assets/8b9ce8f7-77f8-4f79-8192-80430896ae55)
![d5650cbaac6fb65cd757b809d30ee70](https://github.com/user-attachments/assets/69d0685e-8ead-44b2-9579-61a5039e3384)

**Reply by s-hadinger** [reply]
The message `{"ZbZNPReceived":"4F80080200"}` indicates `BDB_COMMISSIONING_FORMATION_FAILURE`

This typically happens if the coordinator detects that the network it tries to create already exists. This happens when you have existing devices still configured to the previous network.

I suggest to change PanID and encryption key with `ZbConfig`

**Reply by vensolezl** [reply]
Thank you for your professional reply, I really did not realize this before, let me try some configuration. By the way, how can these devices be automatically configured when PANids exist in the environment?

**Reply by vensolezl** [reply]
![image](https://github.com/user-attachments/assets/2268f33f-5663-42c4-a263-d0c06657f56d)
I tried the following, first typing ZbConfig, but I got these results.
I then used the ZBconfig command and restarted Tasmota, and the device still didn't start as expected, which was really weird.
![image](https://github.com/user-attachments/assets/00a33d00-6f57-4aa7-9b14-14fee2a77743)
14:11:28.749 CMD: ZbConfig
14:11:28.750 SRC: Serial
14:11:28.751 CMD: Grp 0, Cmd 'ZBCONFIG', Idx 1, Len 0, Pld -99, Data ''
14:11:28.754 RSL: RESULT = {"Command":"Unknown","Input":"ZBCONFIG"}
14:14:18.607 CMD: ZbConfig {"Channel":12,"PanID":"0x1ABC","ExtPanID":"0xDDCCCCCCCCCCCCAA","KeyL":"0xFF0D0B0907050301","KeyH":"0xED0C0A0806040200"}
14:14:18.620 SRC: WebConsole from 10.0.0.171
14:14:18.621 CMD: Grp 0, Cmd 'ZBCONFIG', Idx 1, Len 119, Pld -99, Data '{"Channel":12,"PanID":"0x1ABC","ExtPanID":"0xDDCCCCCCCCCCCCAA","KeyL":"0xFF0D0B0907050301","KeyH":"0xED0C0A0806040200"}'
14:14:18.626 RSL: RESULT = {"Command":"Unknown","Input":"ZBCONFIG {\"Channel\":12,\"PanID\":\"0x1ABC\",\"ExtPanID\":\"0x"}
14:14:32.757 CMD: restart 1
14:14:32.758 SRC: WebConsole from 10.0.0.171
14:14:32.760 CMD: Grp 0, Cmd 'RESTART', Idx 1, Len 1, Pld 1, Data '1'
14:14:32.762 RSL: RESULT = {"Restart":"Restarting"}
00:00:00.002 HDW: ESP32-D0WD-V3 v3.1 
00:00:00.021 UFS: FlashFS mounted with 4400 kB free
00:00:00.035 CFG: Loaded from File, 数量: 25
00:00:00.043 QPC: Count 1
00:00:00.044 CFG: CR 342/699, Busy 0
00:00:00.046 TYA: Active=0
00:00:00.048 ROT: Mode 1
00:00:00.073 CFG: No '*.autoconf' file found
00:00:00.076 BRY: Berry initialized, RAM used 3478 bytes
00:00:00.082 BRY: No 'preinit.be'
00:00:00.085 ZIG: Using Hardware Serial UART2
00:00:00.085 SRC: Restart
00:00:00.087 项目: tasmota - Tasmota 版本 14.3.0(tasmota)-3_1_0(2024-12-10T16:01:47)
00:00:00.088 ZIG: Loading Zigbee data
00:00:00.093 ZIG: No Zigbee device information
00:00:00.096 ZIG: No Zigbee device data
00:00:00.097 ZIG: ZbLoad '<internal_plugin>' loaded successfully
00:00:00.143 BRY: No 'autoexec.be'
00:00:00.279 WIF: 尝试连接...
00:00:01.000 WIF: 连接到 AP1 RAX50 Channel 8 BSSId 74:D0:2B:28:3D:5B 模式: HT40 名称: tasmota-E8FF34-7988...
00:00:01.038 WIF: IPv4 10.0.0.209, mask 255.255.255.0, gateway 10.0.0.1
00:00:02.936 WIF: 已连接
00:00:03.298 RTC: UTC 2024-12-19T13:14:39Z, DST 2024-03-31T02:00:00, STD 2024-10-27T03:00:00
14:14:39.001 RTC: Synced by NTP
14:14:39.011 HTP: Web 服务器地址: tasmota-E8FF34-7988 IP 地址: 10.0.0.209
14:14:39.138 WIF: IPv6 Local fe80::1206:1cff:fee8:ff34%st1
14:14:40.203 RSL: INFO1 = {"Info1":{"Module":"ESP32-DevKit","Version":"14.3.0(tasmota)","FallbackTopic":"cmnd/DVES_E8FF34_fb/","GroupTopic":"cmnd/tasmotas/"}}
14:14:40.206 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Hostname":"tasmota-E8FF34-7988","IPAddress":"10.0.0.209","IP6Global":"","IP6Local":"fe80::1206:1cff:fee8:ff34%st1"}}
14:14:40.207 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":15}}
14:14:42.959 QPC: Reset
14:14:43.436 WIF: IPv6 Global 2001:250:5403:420:1206:1cff:fee8:ff34
14:14:44.953 APP: 启动次数 15
14:14:44.956 RSL: STATE = {"Time":"2024-12-19T14:14:44","Uptime":"0T00:00:09","UptimeSec":9,"Heap":157,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":21,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":40},"Wifi":{"AP":1,"SSId":"RAX50","BSSId":"74:D0:2B:28:3D:5B","Channel":8,"Mode":"HT40","RSSI":100,"Signal":-47,"LinkCount":1,"Downtime":"0T00:00:03"}}
14:14:45.808 CFG: Saved, 数量: 26, 大小: 4096
14:14:51.569 ZIG: rebooting ZNP device
14:14:54.071 ZIG: {"ZbZNPReceived":"4180000201020701"}
14:14:54.072 RSL: RESULT = {"ZbState":{"Status":1,"Message":"CCxxxx ZNP booted","RestartReason":"Power-up","MajorRel":2,"MinorRel":7}}
14:14:54.220 ZIG: checking device configuration
14:14:54.221 ZIG: ZbZNPSent 2102
14:14:54.271 ZIG: {"ZbZNPReceived":"6102020102070146D9340100"}
14:14:54.272 RSL: RESULT = {"ZbState":{"Status":50,"MajorRel":2,"MinorRel":7,"MaintRel":1,"Revision":20240710}}
14:14:54.273 ZIG: ZbZNPSent 2108600000
14:14:54.320 ZIG: {"ZbZNPReceived":"6108000155"}
14:14:54.321 ZIG: ZbZNPSent 2108870000
14:14:54.370 ZIG: {"ZbZNPReceived":"6108000100"}
14:14:54.371 ZIG: ZbZNPSent 2108830000
14:14:54.420 ZIG: {"ZbZNPReceived":"61080002FFFF"}
14:14:54.422 RSL: RESULT = {"ZbState":{"Status":2,"Message":"Resetting configuration"}}
14:14:54.423 ZIG: ZbZNPSent 21090300000103
14:14:54.470 ZIG: {"ZbZNPReceived":"610900"}
14:14:54.471 ZIG: ZbZNPSent 410000
14:14:56.871 ZIG: {"ZbZNPReceived":"4180000201020701"}
14:14:56.872 ZIG: ZbZNPSent 2109830000020100
14:14:56.920 ZIG: {"ZbZNPReceived":"610900"}
14:14:56.921 ZIG: ZbZNPSent 21092D00000800000000CCCCCCCC
14:14:56.970 ZIG: {"ZbZNPReceived":"610900"}
14:14:56.971 ZIG: ZbZNPSent 21098400000400080000
14:14:57.020 ZIG: {"ZbZNPReceived":"610900"}
14:14:57.021 ZIG: ZbZNPSent 21098700000100
14:14:57.070 ZIG: {"ZbZNPReceived":"610900"}
14:14:57.071 ZIG: ZbZNPSent 210962000010000034FF00000000000034FF00000000
14:14:57.120 ZIG: {"ZbZNPReceived":"610900"}
14:14:57.121 ZIG: ZbZNPSent 21096300000100
14:14:57.190 ZIG: {"ZbZNPReceived":"610900"}
14:14:57.192 ZIG: ZbZNPSent 210901010020FFFFFFFFFFFFFFFF5A6967426565416C6C69616E636530390000000000000000
14:14:57.220 ZIG: {"ZbZNPReceived":"610902"}
14:14:57.221 ZIG: ZbZNPSent 21098F00000101
14:14:57.270 ZIG: {"ZbZNPReceived":"610900"}
14:14:57.271 ZIG: ZbZNPSent 2107600001000100
14:14:57.320 ZIG: {"ZbZNPReceived":"610700"}
14:14:57.321 ZIG: ZbZNPSent 21096000000155
14:14:57.370 ZIG: {"ZbZNPReceived":"610900"}
14:14:57.371 ZIG: ZbZNPSent 2108620000
14:14:57.421 ZIG: {"ZbZNPReceived":"61080010000034FF00000000000034FF00000000"}
14:14:57.422 RSL: RESULT = {"ZbState":{"Status":3,"Message":"Configured, starting coordinator"}}
14:14:57.424 ZIG: ZbZNPSent 2F080100080000
14:14:57.470 ZIG: {"ZbZNPReceived":"6F0800"}
14:14:57.471 ZIG: ZbZNPSent 2F080000080000
14:14:57.520 ZIG: {"ZbZNPReceived":"6F0800"}
14:14:57.521 ZIG: ZbZNPSent 2F0504
14:15:00.770 ZIG: {"ZbZNPReceived":"6F0500"}
14:15:00.820 ZIG: {"ZbZNPReceived":"4F80010204"}
14:15:01.020 ZIG: {"ZbZNPReceived":"45C008"}
14:15:03.070 ZIG: {"ZbZNPReceived":"45C008"}
14:15:03.385 ZIG: {"ZbZNPReceived":"45C008"}
14:15:03.920 ZIG: {"ZbZNPReceived":"45C000"}
14:15:03.970 ZIG: {"ZbZNPReceived":"4F80080200"}
14:15:04.020 ZIG: {"ZbZNPReceived":"4F80080200"}
14:15:20.820 ZIG: timeout, goto label 99
14:15:20.821 RSL: RESULT = {"ZbState":{"Status":99,"Message":"Abort"}}
14:15:20.823 ZIG: Abort
14:15:20.823 ZIG: Stopping (99)

**Reply by s-hadinger** [reply]
Hmmm. ZbConfig must be used before the coordinator tries to run, so in the first 15 seconds. This is not ideal, but it's the case for now

**Reply by vensolezl** [reply]
That's not a big problem, at least it solved my problem and confusion, thank you very much, do you have any suggestions on how to automatically generate panID?

**Reply by s-hadinger** [reply]
Why would you generate it automatically? Just take a new random number. It's a one-off.

**Reply by vensolezl** [reply]
Haha, because I am currently researching a project to deploy zigee gateways based on tasmota and use zigbee sensors from aqura in some rooms of a local nursing home, such as door sensor, water immersion, human infrared, vibration and other sensors. In order to realize the regularity of monitoring the daily behavior and activities of the elderly.
The maintainer takes the gateway and plugs it directly into the power supply and network cable, and then triggers the zigbee pairing button to achieve the deployment of the sensor. These people don't have any zigbee control technology, which is why I designed the gateway to use Ethernet.
I plan to deploy 30 gateways, the room location is close to each other, when the deployment will find zigbee can not start normally, these problems have troubled me for a long time.

**Reply by s-hadinger** [reply]
Hi, there is a feature in zigbee coordinators to choose a random unused PanID. But we haven't implemented it in Tasmota, because users generally want more control and prefer to avoid the MCU to do random things out of control.
I would suggest to pre-configure panIDs before deploying the devices while centrally managing the PanID numbering.
You could also force a random PanID generation, but there is always a risk of collision.
